File: app/src/main/java/com/example/recyclerview/entity/ImageInfoBean.java
package com.example.recyclerview.entity;

public class ImageInfoBean implements java.io.Serializable{
    public long id;

    public String name;
    public long size;
    public String path;

    @Override
    public String toString() {
        return "ImageInfo{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", size=" + size +
                ", path='" + path + '\'' +
                '}';
    }

    public ImageInfoBean(long id, String name, long size, String path) {
        this.id = id;
        this.name = name;
        this.size = size;
        this.path = path;
    }

    public ImageInfoBean() {
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public long getSize() {
        return size;
    }

    public void setSize(long size) {
        this.size = size;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }
}

File: app/src/main/java/com/hanlin/image_editor_hanlin/entity/ImageInfoBean.kt
package com.hanlin.image_editor_hanlin.entity

import java.io.Serializable

class ImageInfoBean : Serializable {
    @JvmField
    var id: Long = 0

    @JvmField
    var name: String? = null
    @JvmField
    var size: Long = 0
    @JvmField
    var path: String? = null

    override fun toString(): String {
        return "ImageInfo{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", size=" + size +
                ", path='" + path + '\'' +
                '}'
    }

    constructor(id: Long, name: String?, size: Long, path: String?) {
        this.id = id
        this.name = name
        this.size = size
        this.path = path
    }

    constructor()
}
